### ISSUE TITLE
Test Theme ID == null

### ISSUE THREAD

**Issue by ZoeJa** [pixelcabin/shopify_theme_integration_tests#2]
I am running through the instructions in the README file, I have successfully got the Cypress working and linked up with Travis CI. However, I am getting an error when running the code with Travis CI. The code is exiting due to Test theme ID being null. 
Any help with why this is happening and if there is something I can do to solve it would be much appreciated. 
Thanks.

Here is the end of the log in Travis:
![travisCILog](https://user-images.githubusercontent.com/49906033/62057905-654cc500-b218-11e9-9027-2c583fe43fad.png)

I have provided it with the correct API keys and the correct ngrok auth token. I have also double checked that the private app in my store has read and write permissions.

EDIT:
I solved this by adding: 
npm install grunt-cli -g 
to the before_script section in the .travis.yml file.

**Comment by michaelrshannon** [comment]
Hi @ZoeJa apologies for the slow reply - I had missed this issue when you opened it a couple of weeks ago. I haven't seen this problem present as part of my own testing, but I'll try and recreate and update the repo as necessary - thanks for raising!

**Comment by leighs-hammer** [comment]
Yup I noticed that as well @ZoeJa, I will PR that back in, it's just missing grunt when it calls zip.

**Comment by leighs-hammer** [comment]
Should be fixed in the follow the above listed PR as far as the initial issue.
